src/ssmDocument/ssm/ssmClient.ts

```diff
@@ -128,7 +122,7 @@ export async function activate(extensionContext: ExtensionContext) {
     })
 }
 
-function getSettings(): Settings {
+function getSettings() {
```

Comment:
Why remove the return type?

Reply:
I removed the file where this interface was defined. Will define the interface somewhere else and add the return type back.

Reply:
You don't need to have an interface if you're only using it in one place, but you should have a return type. Just do something like:
```
function getSettings(): {
    setting1: string,
    setting2: string,
} {
... function goes here ... 
}
```